Library/Formula/jsawk.rb

```diff
@@ -1,16 +1,18 @@
-require 'formula'
-
 class Jsawk < Formula
-  homepage 'https://github.com/micha/jsawk'
-  url 'https://github.com/micha/jsawk/archive/1.4.tar.gz'
-  sha1 '4f2c962c8a5209764116457682985854400cbf24'
+  homepage "https://github.com/micha/jsawk"
+  url "https://github.com/micha/jsawk/archive/1.4.tar.gz"
+  sha1 "4f2c962c8a5209764116457682985854400cbf24"
 
-  head 'https://github.com/micha/jsawk.git'
+  head "https://github.com/micha/jsawk.git"
 
-  depends_on 'spidermonkey'
+  depends_on "spidermonkey"
 
   def install
-    mv "README.markdown", "README"
```

Comment:
Sorry for highjack this PR. But a quick question, do we want to extend the `ext` list. OR just ignore the `ext` at all so `README.whatever` can be installed?